compose/material3/material3/src/skikoMain/kotlin/androidx/compose/material3/WideNavigationRail.skiko.kt

```diff
@@ -24,7 +24,6 @@ import androidx.compose.ui.window.Dialog
 import androidx.compose.ui.window.DialogProperties
 
 @Immutable
-@ExperimentalMaterial3ExpressiveApi
```

Comment:
Probably api dump will fail, as it should add a new non experimental API there

Reply:
> 1.10 (build buildSrc already handles it)

My plan wasn't included to migrate to the new API checks in this migration. We can do that, but step by step